packages/react-sdk/src/components/ElementBar/ColorPickerButton/ElementColorPicker.tsx

```diff
@@ -45,7 +45,9 @@ export function ElementColorPicker() {
 
   const hasShape = activeElements.some((e) => e.type === 'shape');
   const hasOther = activeElements.some((e) => e.type !== 'shape');
-  const hasOnlyImages = activeElements.every((e) => e.type === 'image');
+  const hasOnlyNonColoredElements = activeElements.every(
+    (e) => e.type !== 'shape' && e.type !== 'path',
+  );
```

Comment:
Why was this changed from images to non-colored elements?

Reply:
This is only used here https://github.com/nordeck/matrix-neoboard/pull/771/files#diff-16237f46fd34aa510aa49dc62dcedba6050303bacc275251c5523bf2d36e3e4fR93 . If there is nothing selected, that has a colour, then do nothing. So instead of `hasOnlyImagesOrFramesOrWhateverComesInTheFuture` I think that what we want to have here is without colour.